src/assets/scripts/client/constants/themes/default/modules/radarTarget.js

```diff
@@ -24,7 +24,7 @@ export const RADAR_TARGET_THEME = {
      * @memberof RADAR_TARGET_THEME
      * @property HISTORY_DOT_OUTSIDE_RANGE
      */
-    HISTORY_DOT_OUTSIDE_RANGE: COLOR.WHITE,
+    HISTORY_DOT_OUTSIDE_RANGE: COLOR.WHITE_05,
```

Comment:
History dots must be the same color at all times. The higher transparency `WHITE_05` is a good choice, but we must set it as _both_ the `inside_range` and `outside_range` color.

Reply:
done